tools/probes/probes_main.c

```diff
@@ -234,16 +245,17 @@ void parse_data(char *file_in)
 		fclose(fd_in);
 		exit(0);
 	}
-	memset(&data, 0, sizeof(uint32_t) * DATA_READ_LIMIT);
+	memset(&data, 0, DATA_READ_LIMIT);
```

Comment:
love C. I guess here both `memset(&data...)` and `memset(data...)` would work and mean the same